extensions/2.0/Vendor/EXT_materials_clearcoat_color/README.md

```diff
@@ -0,0 +1,131 @@
+
+As with the core glTF 2.0 spec, all coloured textures are assumed to be sRGB and all coloured factors are assumed to be linear.
+
+|                                  | Type                                                                            | Description                            | Required             |
+|----------------------------------|---------------------------------------------------------------------------------|----------------------------------------|----------------------|
+|**clearcoatColorFactor**    | `number[3]`                                                                                       | The clearcoat's colour factor.  | No, default: `[1.0, 1.0, 1.0]`   |
+|**clearcoatColorTexture**   | [`textureInfo`](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#reference-textureinfo)  | The clearcoat layer's color texture.  | No                               |
+
+As with other factor/texture combinations, these two values are multiplied before being used in calculations. If not defined, the defaults represent a white transmission color and, therefore, no absorption.
+```
+clearcoat_transmission_color = clearcoatColorFactor * clearcoatColorTexture
+```
+
+## Physical Interpretation
+
+The clearcoat is a thin, dielectric layer that specularly-transmits light through it but, by default, doesn't absorb light. This extension allows us to specify the observed color tint of light passing through the coating from the underlying layer. The `clearcoat_transmission_color` represents the **square of transmittance of the coating**. This is because the observed color tint of the underlying material results after light has passed through the coating twice (in incoming and outgoing directions), making the final color more easily artist-directable.
+
+### View-Dependent Absorption
+
+Light rays travel different distances through the clearcoat layer depending on the angle of incidence. At grazing angles (when viewing the surface nearly edge-on), light travels a longer distance through the clearcoat medium, resulting in stronger absorption and more pronounced color saturation. Conversely, at normal incidence (when viewing the surface straight-on), light travels the shortest path through the clearcoat, producing less absorption. The `clearcoat_transmission_color` represents the color observed at normal incidence.
+
+Note that this view-dependent absorption is more pronouced when the ratio of IOR between the coating and surounding medium (usually air) is low. This is because, at higher IOR's, light impacting the coating at grazing angles gets refracted into the coating more strongly, shortening the path length through the coat and decreasing the absorption.
```

Comment:
```suggestion
Note that this view-dependent absorption is more pronounced when the ratio of IOR between the coating and surrounding medium (usually air) is low. This is because, at higher IOR's, light impacting the coating at grazing angles gets refracted into the coating more strongly, shortening the path length through the coat and decreasing the absorption.
```